provider/pkg/provider/deployment_settings.go

```diff
@@ -0,0 +1,585 @@
+package provider
+
+import (
+	"context"
+	"fmt"
+	"path"
+
+	pbempty "github.com/golang/protobuf/ptypes/empty"
+	"github.com/pulumi/pulumi-pulumiservice/provider/pkg/internal/pulumiapi"
+	"github.com/pulumi/pulumi/sdk/v3/go/common/apitype"
+	"github.com/pulumi/pulumi/sdk/v3/go/common/resource"
+	"github.com/pulumi/pulumi/sdk/v3/go/common/resource/plugin"
+	pulumirpc "github.com/pulumi/pulumi/sdk/v3/proto/go"
+)
+
+type PulumiServiceDeploymentSettingsInput struct {
+	pulumiapi.DeploymentSettings
+	Stack pulumiapi.StackName
+}
+
+const FixMe = "FIX ME"
```

Comment:
Probably want something more direct than `FIX ME` for this.

Reply:
Do you have a suggestion?

Reply:
I think anything that indicates that the value needs to be replaced and why would work. Maybe `<value is secret and must be replaced>` or something similar?